net/dag.go

```diff
@@ -125,44 +123,28 @@ func (p *Peer) dagWorker(jobs chan *dagJob) {
 		default:
 		}
 
-		children, err := p.processLog(
-			p.ctx,
-			job.txn,
-			job.collection,
-			job.dsKey,
-			job.fieldName,
-			job.node,
-			job.nodeGetter,
-			true,
-		)
-		if err != nil {
-			log.ErrorE(
-				p.ctx,
-				"Error processing log",
-				err,
-				logging.NewKV("Datastore key", job.dsKey),
-				logging.NewKV("CID", job.node.Cid()),
-			)
-			job.session.Done()
-			continue
-		}
-
-		if len(children) == 0 {
-			job.session.Done()
-			continue
-		}
-
 		go func(j *dagJob) {
-			p.handleChildBlocks(
-				j.session,
-				j.txn,
-				j.collection,
-				j.dsKey,
-				j.fieldName,
-				j.node,
-				children,
-				j.nodeGetter,
-			)
+			if j.nodeGetter != nil && j.cid.Defined() {
+				cNode, err := j.nodeGetter.Get(p.ctx, j.cid)
+				if err != nil {
+					log.ErrorE(p.ctx, "Failed to get node", err, logging.NewKV("CID", j.cid))
```

Comment:
todo: Please define these errors (and others in other files) in an errors.go file

Reply:
They are not returned errors. It's a log message. We don't predefine our log messages at all currently.

Reply:
Our log messages are public, just like errors.  Most of the time the error messages will be exposed via logs (e.g. CLI/HTTP), the reasons for housing them in a dedicated and standardised file are exactly the same.

Reply:
Although there is no value in defining them in the errors.go file. It's not an error that can be handled or compared. It's just a message in a log file.

Reply:
Time for a log.go file :,)

Reply:
I'm ok with a solution like that. I'll leave it as is for now and create a separate PR to take care of log messages.

Reply:
> It's just a message in a log file.

nitpick: It is an error message in a log file, and indistinguishable to our users from any other 'kind' of error.

> I'll leave it as is for now and create a separate PR to take care of log messages

I'm fine with that, but I do think they should live in an `errors.go` file, because they are errors.